GH_Lettuce/3rd_AutonomousGreenhouseChallenge_1stExperiment/README.md

```diff
@@ -0,0 +1,117 @@
+# **International Autonomous Greenhouse Challenge - 3rd edition - 1st experiment**
```

Comment:
Can't use WUR's language. Make it Koidra authentic. It doesn't need to be verbose.

For folder names, use this pattern: `Koidra.AGC2022.1` and `Koidra.AGC2022.2`

Reply:
Also add a few public press releases that highlight Koidra's success.
- https://www.wur.nl/en/newsarticle/familiar-face-leads-team-koala-to-win-the-autonomous-greenhouse-challenge-for-a-second-time.htm
- https://www.geekwire.com/2022/how-a-former-microsoft-researcher-used-ai-to-grow-award-winning-lettuce-from-5000-miles-away/